save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ContestGlobalRatingView.kt

```diff
@@ -0,0 +1,230 @@
+                                src = cellProps.row.original.avatar?.let {
+                                    "/api/$v1/avatar$it"
+                                }
+                                    ?: run {
+                                        "img//company.svg"
+                                    }
+                                style = jso {
+                                    height = 2.rem
+                                    width = 2.rem
+                                }
+                            }
+                            href = "#/${cellProps.value}"
+                            +" ${cellProps.value}"
+                        }
+                    }
+                }
+            }
+            column(id = "rating", header = "Rating") {
+                Fragment.create {
+                    td {
+                        +"4560"
+                    }
+                }
+            }
+        },
+        isInvisibleTable = true,
+        useServerPaging = false,
+        usePageSelection = false,
+        getAdditionalDependencies = {
+            arrayOf(it)
+        }
+    )
+    private val renderingProjectChampionsTable = tableComponent(
+        columns = columns<Project> {
+            column(id = "index", header = "Position") {
+                Fragment.create {
+                    td {
+                        val index = it.row.index + 1 + it.state.pageIndex * it.state.pageSize
+                        +index
+                    }
+                }
+            }
+            column(id = "name", header = "Name", { name }) { cellProps ->
+                Fragment.create {
+                    td {
+                        a {
+                            href = "#/${cellProps.row.original.organization.name}/${cellProps.value}"
+                            +" ${cellProps.value}"
+                        }
+                    }
+                }
+            }
+            column(id = "rating", header = "Rating") {
+                Fragment.create {
+                    td {
+                        +"1370"
+                    }
+                }
+            }
+        },
+        isInvisibleTable = true,
+        useServerPaging = false,
+        usePageSelection = false,
+        getAdditionalDependencies = {
+            arrayOf(it)
+        }
+    )
+
+    init {
+        state.organizations = emptyArray()
+        state.projects = emptyArray()
+        state.selectedTab = UserRatingTab.ORGS
+    }
+
+    override fun componentDidMount() {
+        super.componentDidMount()
+
+        scope.launch {
+            val organizationsFromBackend: List<Organization> = get(
+                url = "$apiUrl/organizations/all",
+                headers = jsonHeaders,
+                loadingHandler = ::classLoadingHandler,
+            )
+                .decodeFromJsonString()
+
+            val projectsFromBackend: List<Project> = get(
+                url = "$apiUrl/projects/all",
```

Comment:
```suggestion
                url = "$apiUrl/projects/not-deleted",
```
?

Reply:
I'm not sure. Perhaps we can show here all the projects that have ever existed. At ContestView we do the same

Reply:
I think we should at least mark inactive users/projects/organizations, for example make their icons fading gray